include/ArcadeStruct.hpp

```diff
@@ -0,0 +1,24 @@
+/*
+** EPITECH PROJECT, 2023
+** Arcade
+** File description:
+** Useful struct for all the project
+*/
+
+#pragma once
+
+namespace Arcade {
+    struct Vector2f {
```

Comment:
Why is z here for a vector2f ?

Reply:
For have only 1 struct to represent pos or size in 2d and 3d, if its 2d z is always 0